myhome/materials/urls.py

```diff
@@ -0,0 +1,18 @@
+from django.urls import path
+from rest_framework.routers import DefaultRouter
+
+from .apps import MaterialsConfig
+from .views import CourseViewSet, LessonCreateAPIView, LessonListAPIView, LessonRetrieveAPIVew, LessonUpdateAPIView, LessonDestroyAPIView
+
+app_name = MaterialsConfig.name
+
+router = DefaultRouter ()
+router.register ( r'course', CourseViewSet, basename='course' )
+
+urlpatterns = [
+                  path ( 'lesson/create/', LessonCreateAPIView.as_view (), name='lesson_create' ),
+                  path ( 'lesson/', LessonListAPIView.as_view (), name='lesson_list' ),
+                  path ( 'lesson/<int:pk>/', LessonRetrieveAPIVew.as_view (), name='lesson_detail' ),
+                  path ( 'lesson/update/<int:pk>/', LessonUpdateAPIView.as_view (), name='lesson_update' ),
```

Comment:
по rest-api должно быть так
```suggestion
                  path ( 'lesson/<int:pk>/update/', LessonUpdateAPIView.as_view (), name='lesson_update' ),
```